Add some new town names

Strongis13 opened the pull request:
Added town names for HOTS battlegrounds and some missing WoW locations. Also removed some of the more obscure WoW names that weren't very significant

Strongis13 committed:
Added maps from HOTS and some new WoW locations

Strongis13 committed:
remove obscure wow names, and add some missing ones

castle-d commented on empiretownnames.nml:
```diff
@@ -346,25 +345,14 @@ town_names(wow) {
-    text("Sylvanaar", 1),
     text("Ogri’la", 1),
-    text("Thunderlord Stronghold", 1),
-    text("Area 52", 1),
```
No.

Strongis13 replied:
Reverted

castle-d commented on empiretownnames.nml:
```diff
@@ -745,7 +724,8 @@ town_names {
     town_names(world, 1),
-    town_names(statecap, 1)
+    town_names(statecap, 1),
+    town_names(hots, 1)
```
make this hots, 2
```suggestion
    town_names(hots, 2)

```

castle-d commented on empiretownnames.nml:
```diff
@@ -745,7 +724,8 @@ town_names {
     town_names(world, 1),
-    town_names(statecap, 1)
+    town_names(statecap, 1),
+    town_names(hots, 1)
```
Can make HOTS names more common since there's so few
```suggestion
    town_names(hots, 2)
```

Strongis13 replied:
Updated this in most recent commit

castle-d reviewed (changes requested):
Please fix these

Strongis13 committed:
Changes requested by upstream

Re-added Area 52 as a possible name, and increased spawn change of HOTS name category
